en/docs/WIP/config-catalog.md

```diff
@@ -1097,7 +1155,7 @@ https_endpoint = "https://localhost:${https.nio.port}"</code></pre>
 
                                     </div>
                                     <div class="param-description">
-                                        <p>ex: ws://localhost:9099, [Same as HTTP]</p>
+                                        <p>E.g., ws://localhost:9099</p>
```

Comment:
E.g., should be e.g.
Please make this change in all other places in this PR as well.

Reply:
I have changed all occurrences.